From 0ec3bc92dd6dcc5a145248b5866e7d859c571ceb Mon Sep 17 00:00:00 2001
From: "Prashanth D. Rao" <prashanth.chengi@gmail.com>
Date: Sun, 17 May 2026 02:01:58 +0200
Subject: [PATCH] extend connection limit

---
 Common/sources/constants.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Common/sources/constants.js b/Common/sources/constants.js
index 55c25026..550d5e80 100644
--- a/Common/sources/constants.js
+++ b/Common/sources/constants.js
@@ -88,7 +88,7 @@ exports.LICENSE_RESULT = {
   NotBefore: 16
 };
 
-exports.LICENSE_CONNECTIONS = 20;
+exports.LICENSE_CONNECTIONS = 10000;
 exports.LICENSE_USERS = 3;
 exports.LICENSE_EXPIRE_USERS_ONE_DAY = 24 * 60 * 60; // day in seconds
 
-- 
2.54.0

